File: AQ_LIB/resources/api/PythonExamples/GridDisplay/GridDisplay.py
#Import the MLIB CLIENT API
import MLIB_CLIENT_API as m

#Load Calendars and Config Files
print ( m.setUpMLIB("","") )

#Get the MLIB Version
print ( m.meUtilityVersion() )


print ( m.meLWOLoad("C://EURYC_EUR3ML.JSON") )

#Get Curve Inputs
print ("Curve Inputs")
print ( m.meLWOCurveDisplay("EURYC_EUR3ML") ) # Use Handle Name

# print ( m.meLWOGridDisplay("EURYC_EUR3ML", False) ) # Use Handle Name
print ( m.meLWOGridDisplay("EURYC_EUR3ML") ) # Use Handle Name

#Get Pseudo Discount Factor(s)
print ( "Psuedo Discount Factors" )
print ( m.meCurveResultsDisplayDiscountFactors("EURYC","EUR3ML") ) # Curve Collection, Curve Index

File: AQ_LIB/resources/api/RExamples/GridDisplay/GridDisplay.R
# MLIB test yield curnve 

#   E:/MLIB/MLIBQ_DEV1/target2017/32/ReleaseR/MLIB_CLIENT_API/out/DLL

mlib_test_yc<-function(){
  
  mlib_source_lib = "E:/MLIB/MLIBQ_DEV1/target2017/32/ReleaseR/MLIB_CLIENT_API/out/DLL/MLIB_CLIENT_API.dll"
  mlib_source = "E:/MLIB/MLIBQ_DEV1/target2017/32/ReleaseR/MLIB_CLIENT_API/out/DLL/MLIB_CLIENT_API.R"
  dyn.load(mlib_source_lib)
  source(mlib_source)
  
  setUpMLIB("","")
  
  meUtilityVersion()
  
  mlib_curve = meLWOLoad("E:/MLIB/MLIBQ_DEV1/target2017/32/ReleaseR/EURYC_EUR3ML.JSON")
  mlib_curve
  
  meLWOGridDisplay(mlib_curve)
  

}